src/main/java/com/github/sardine/ant/command/Put.java

```diff
@@ -39,6 +48,13 @@
 	/** The optional content type of the single source file. */
 	private String fContentType;
 
+	/** Settings for put with filtering. */
+	private FilterSetCollection fFilterSets = new FilterSetCollection();
+	private Vector<FilterChain> fFilterChains = new Vector<FilterChain>();
+	private String fInputEncoding=null;
+	private String fOutputEncoding=null;
+	private boolean fInMemoryFiltering = true;
```

Comment:
A bad habbit due to my companys style guide: 'f' for 'field' has to be before all class properties. If you don't like it i drop it.